### PR TITLE
Polyline fixes

### PR DESCRIPTION
**Pull request by mattiskoh**
### What type of change is this?

- [x ] Bug fix in a **backwards-compatible** manner.

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ x] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ x] I ran lint on my computer and there are no errors (i.e. `invoke lint`).

### CODE REVIEW

File: src/compas/geometry/primitives/polyline.py [hunk 1]
```diff
@@ -340,7 +340,7 @@ def divide_polyline(self, num_segments):
 
         return self.divide_polyline_by_length(segment_length, False)
 
-    def divide_polyline_by_length(self, length, strict=True):
+    def divide_polyline_by_length(self, length, strict=True, tol=1e-06):
```
Comment (beverlylytle) [inline src/compas/geometry/primitives/polyline.py:343]: Add `tol` to the parameters list in the docstring, please...

**Review by beverlylytle — APPROVED** [review]
Looks good!  Just the one comment to address, and then we merge!

File: src/compas/geometry/primitives/polyline.py (hunk 1 above)
Comment (beverlylytle) [reply src/compas/geometry/primitives/polyline.py:343]: oh, also, there shouldn't be blank lines between the parameters... it makes the docs look funny... see https://compas.dev/compas/latest/api/generated/compas.geometry.Polyline.divide_polyline_by_length.html#compas.geometry.Polyline.divide_polyline_by_length

Comment (mattiskoh) [reply src/compas/geometry/primitives/polyline.py:343]: just fyi, in `offset_polyline` tol isn't referenced!  https://github.com/compas-dev/compas/blob/be558946aaf34b30dec0cdf4d2e52d805a449bf5/src/compas/geometry/offset/offset.py#L125


Will make the changes though!

File: src/compas/geometry/primitives/polyline.py [hunk 2]
```diff
@@ -340,16 +340,19 @@ def divide_polyline(self, num_segments):
...
 
-    def divide_polyline_by_length(self, length, strict=True):
-        """Splits a polyline in segments of a given length
+    def divide_polyline_by_length(self, length, strict=True, tol=1e-06):
+        """Splits a polyline in segments of a given length.
 
-        Parameters:
+        Parameters
         -----------
         length : float
```
Comment (beverlylytle) [inline src/compas/geometry/primitives/polyline.py:349]:
```suggestion
```

File: src/compas/geometry/primitives/polyline.py [hunk 3]
```diff
@@ -340,16 +340,19 @@ def divide_polyline(self, num_segments):
...
 
         strict : bool
             If set to ``False``, the remainder segment will be added even if it is smaller than the desired length
```
Comment (beverlylytle) [inline src/compas/geometry/primitives/polyline.py:352]:
```suggestion
```

**Pull request merged**